Zilon.Core/Zilon.Core/Tactics/StaticObject.cs

```diff
@@ -62,23 +61,23 @@ private bool GetIsMapBlock()
 
         private Type GetSanitizedModuleType(Type type)
         {
-            var moduleAttribute = type.GetCustomAttributes(typeof(StaticObjectModuleAttribute), false);
-            if (moduleAttribute is null)
+            var implementedInterfaces = type.GetInterfaces();
+
+            foreach (var implementedInterface in implementedInterfaces)
             {
-                var parent = type.BaseType;
-                if (parent is null)
+                //var moduleAttribute = implementedInterface.GetCustomAttributes(typeof(StaticObjectModuleAttribute), false);
+                var moduleAttribute = implementedInterface.CustomAttributes.SingleOrDefault(x => x.AttributeType == typeof(StaticObjectModuleAttribute));
+                if (moduleAttribute is null)
                 {
-                    return null;
+                    continue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant jump.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5368096)